### PR TITLE
feat(tile-converter): add i3s attributes worker

### PR DESCRIPTION
**Pull request by dryabinin-actionengine**

### CODE REVIEW

File: modules/tile-converter/src/i3s-converter/helpers/i3s-attribute-transformation.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+export type AttributesTransformationOptions = WriterOptions & {
+  attributesTransformations?: AttributesOptions & {
+    useCartesianPostions: false;
+  };
+};
+
+/**
+ * Transform B3DM data to I3S attributes
+ */
+export const I3SAttributeTransformation: Writer = {
```
Comment (belom88) [inline modules/tile-converter/src/i3s-converter/helpers/i3s-attribute-transformation.ts:23]: I don't think it is a writer. It should be just worker like in `compression` or `crypto` modules:
"modules/compression/src/compression-worker.ts"

**Review by belom88 — CHANGES_REQUESTED** [review]
We don't need a writer in my opinion. You can call `convertAttributes` directly from the worker.

File: modules/tile-converter/src/i3s-converter/helpers/i3s-attribute-transformation.ts (hunk 1 above)
Comment (dryabinin-actionengine) [reply modules/tile-converter/src/i3s-converter/helpers/i3s-attribute-transformation.ts]: Done

**Review by ibgreen — APPROVED** [review]
Are you able to add any benchmarks to test the performance?

**Comment by dryabinin-actionengine** [comment]
> Are you able to add any benchmarks to test the performance?

Possibly we can do it as separate PR.

**Review by belom88 — APPROVED** [review]

**Pull request merged**